dist/ion-gallery.js

```diff
@@ -42,6 +43,8 @@
           }
         });
       }());
+
+      $scope.customCallback = typeof $scope.ionItemCallback !== 'undefined';
```

Comment:
Makes sense, the ? symbol makes it an optional parameter. Give it a quick go and then I'll merge it. Good work :+1: